### PR TITLE
#2699 created models for FAQs and Milestone

### LINKED ISSUE #2699: [Recruitment] - Create model for FAQ and Milestone
### Desired Changes

We need a way to store information about FAQs and Milestones on the Application page. We will create two models for these which we will call FAQ’s and Milestone. You will have to create a migration for this by running `yarn prisma migrate`

Model FrequentlyAskedQuestion
String faqId
String question
String answer
User userCreated
String userCreatedId
User userDeleted
String userDeletedId
Date dateCreated
Date dateDeleted

Model Milestone
String milestoneId
Date dateOfEvent
String name
String description?
User userCreated
String userCreatedId
User userDeleted
String userDeletedId
Date dateCreated
Date dateDeleted

### Screenshots (as needed)

_No response_

### PR DESCRIPTION
**Pull request by Aaryan1203**
## Changes
- Created models for FAQs and Milestones
- Created shared types for FAQs and milestones

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #2699

### CODE REVIEW

File: src/backend/src/prisma/schema.prisma [hunk 1]
```diff
@@ -874,3 +878,28 @@ model Organization {
...
+model FrequentlyAskedQuestion {
+  frequentlyAskedQuestionId String    @id @default(uuid())
+  question                  String
+  answer                    String
+  userCreated               User      @relation(fields: [userCreatedId], references: [userId], name: "frequentlyAskedQuestionCreator")
+  userCreatedId             String
+  userDeleted               User?     @relation(fields: [userDeletedId], references: [userId], name: "frequentlyAskedQuestionDeleter")
+  userDeletedId             String?
+  dateCreated               DateTime  @default(now())
+  dateDeleted               DateTime?
```
Comment (Peyton-McKee) [inline src/backend/src/prisma/schema.prisma:893]: these need to be linked to an organization

File: src/backend/src/prisma/schema.prisma [hunk 2]
```diff
@@ -874,3 +878,28 @@ model Organization {
...
+}
+
+model Milestone {
+  milestoneId   String    @id @default(uuid())
```
Comment (Peyton-McKee) [inline src/backend/src/prisma/schema.prisma]: needs to be linked to an organization

File: src/backend/src/prisma/schema.prisma [hunk 3]
```diff
@@ -874,3 +878,28 @@ model Organization {
...
+  name          String
+  dateOfEvent   DateTime
+  description   String?
```
Comment (Peyton-McKee) [inline src/backend/src/prisma/schema.prisma]: I dont think this needs to be optional, we can just have it be an empty string

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Looks good

**Review by walker-sean — APPROVED** [review]

**Pull request merged**